deploy/vagrant/create

```diff
@@ -103,15 +87,11 @@ vagrant provision
 
 # show master IP
 if [[ "$?" == "0" ]]; then
- HERE=`dirname $0`
- pushd $HERE > /dev/null
  if [[ "$PROVIDER" == "aws" ]]; then
-  master=`cat ${HERE}/.vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory | grep Master | cut -d' ' -f2 | cut -d'=' -f2`
+  master=`cat .vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory | grep Master | cut -d' ' -f2 | cut -d'=' -f2`
  else
-  master=`tail -n 1 $HERE/files/hosts | cut -d' ' -f1`
+  master=`tail -n 1 files/hosts | cut -d' ' -f1`
```

Comment:
Why do we switch from absolute path to relative path? Seems more prone to error?